pyproject.toml

```diff
@@ -1,3 +1,12 @@
+[tool.djlint]
+blank_line_after_tag="load,extends,import,from"
+blank_line_before_tag="load,extends,block"
+max_line_length=120
+# _base.html excluded because of the bad </script> nesting after format
+exclude=".templates/layouts/_base.html"
```

Comment:
Disable djlint for the script bit instead of the whole template?.

Reply:
It's because we are not able to fix the formatting of this file, it flattens the </script> tags no matter what config settings we use, if we are happy with incorrect nesting of <script> I can remove this exclusion.

Reply:
Looks like a bug in djlint as it is ignore the djlint on/off comments.

Reply:
Yeah, I am looking through issues, should be there as an existing one.

Reply:
It is an existing issue, the `</script>` indentation.